libs/auth/src/interfaces/auth-facade.interface.ts

```diff
@@ -1,14 +0,0 @@
-import { Observable } from 'rxjs';
-import { Action } from '@ngrx/store';
-
-import { DaffStoreFacade } from '@daffodil/core';
-import { DaffAuthToken } from '../models/auth-token';
-
-export interface DaffAuthFacadeInterface<T extends DaffAuthToken> extends DaffStoreFacade<Action> {
```

Comment:
Is this interface superseded by another? Why is it removed?

Reply:
Honestly this branch is so old I don't remember. Its not superseded. Should I add it back?

Reply:
It'll be needed for the testing facades, I'll add it back.

Reply:
added